common/utils.go

```diff
@@ -21,3 +32,61 @@ func StdLang(lang string) string {
 	return LANG_UNKNOWN
 }
 
+func CreateCUTypeSource(source *models.Source, mdb boil.Executor) (*models.ContentUnit, error) {
```

Comment:
This is far from being common. Please remove 
Common is as clean as possible a package.

Reply:
fix